packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -96,23 +93,24 @@ export const makeBidSpecShape = (currencyBrand, collateralBrand) => {
 /**
  * @typedef {object} BookDataNotification
  *
- * @property {PriceQuote    | null}      startPrice identifies the priceAuthority and price
- * @property {PriceLevel    | null}      currentPriceLevel the price at the current auction tier
+ * @property {Ratio         | null}      startPrice identifies the priceAuthority and price
+ * @property {Ratio         | null}      currentPriceLevel the price at the current auction tier
  * @property {Amount<'nat'> | null}      startProceedsGoal The proceeds the sellers were targeting to raise
  * @property {Amount<'nat'> | null}      remainingProceedsGoal The remainder of
  *     the proceeds the sellers were targeting to raise
  * @property {Amount<'nat'> | undefined} proceedsRaised The proceeds raised so far in the auction
  * @property {Amount<'nat'>}             startCollateral How much collateral was
  *    available for sale at the start. (If more is deposited later, it'll be
  *    added in.)
- * @property {Amount<'nat'> | undefined} collateralAvailable The amount of collateral remaining
+ * @property {Amount<'nat'> | null}      collateralAvailable The amount of collateral remaining
  */
 
 /**
  * @param {Baggage} baggage
  * @param {ZCF} zcf
+ * @param {import('@agoric/zoe/src/contractSupport/recorder.js').MakeRecorderKit} makeRecorderKit
```

Comment:
I'm not happy about these deep imports but I think the real solution is to have a new package for contract support. such that this would be,
```suggestion
 * @param {import('@agoric/contracts').MakeRecorderKit} makeRecorderKit
```

Reply:
concur.

Reply:
queued up: https://github.com/Agoric/agoric-sdk/issues/7392